python/ray/train/_internal/storage.py

```diff
@@ -36,12 +36,25 @@
 
 logger = logging.getLogger(__file__)
 
+ARTIFACT_STORAGE_PATH_ENV_VARS = [
+    "RAY_ARTIFACT_STORAGE",
+    "ANYSCALE_ARTIFACT_STORAGE",
```

Comment:
For simplicity can we have a single Ray environment variable and just set that in Anyscale?

Reply:
So `ANYSCALE_ARTIFACT_STORAGE` is already being set. I've added the `RAY_` variable to make it semantically sensible for pure OSS users. Happy to converge to only one, but I believe the anyscale product will continue to use ANYSCALE_ prefixed variables.

cc @anyscalesam wdyt?

Reply:
Ah so right now the proposal in this PR is to read both environment variables in Ray, and set one in Anyscale. My proposal is to switch it so we only read one (Ray) environment variable in Ray, and set both in Anyscale.

Reply:
Got you - I guess the missing piece here is that `ANYSCALE_ARTIFACT_STORAGE` already exists and is being set on Anyscale. So if we read both (I think it's fine to only document one) we don't need more changes on the anyscale side.

If changes on anyscale side are acceptable, I would also prefer the `RAY_` variable.

Reply:
The proposal sounds good.